schemas/data/average-minute-audience.schema.json

```diff
@@ -0,0 +1,56 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/data/average-minute-audience",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Average Minute Audience",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/data/metricdefinition"
+  ],
+  "description": "Describes the average content time spent for a specific media item - i.e. the total content time spent divided by the length for all of the playback sessions.",
+  "definitions": {
+    "metricdefinition": {
+      "properties": {
+        "@id": {
+          "type": "string",
+          "const": "xdm:averageMinuteAudience",
+          "description": "The unique identifier of this metric."
+        },
+        "schema:name": {
+          "type": "string",
+          "const": "metric: averageMinuteAudience",
+          "description": "Describes the average content time spent for a specific media item - i.e. the total content time spent divided by the length for all of the playback sessions."
+        },
+        "xdm:measurement": {
```

Comment:
looks like more copy/paste and then incomplete editing

Reply:
Taken out this file as it was using the legacy metricdefinition. Using measure instead.